beginners-yoga-course.html

```diff
@@ -22,14 +22,13 @@
 
 <!-- HEADER -->
 <div class="pageHeader__logo"></div>
-<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
```

Comment:
@smellular To clarify why I assume you were having trouble fixing this: the `.gridCell--row` styles are only `display: flex` [in a media query](https://github.com/reneesharpless/reneesharpless.github.io/blob/master/main.css#L73-L75), so on the mobile version of the site this was not `display: flex` which made the `.gridCell--row--justifyCenter` class not work as expected. This is why I removed this node and moved the styles over to `.pageHeader`, because we want it to behave a bit differently from the other grid stuff at this time.